File: CPP/Arrays/03.ZeroesOnes.cpp
#include <bits/stdc++.h>
using namespace std;

void zeroOne(int arr[], int size){

    int zeroCount = 0;
    int oneCount = 0;

    for (int i = 0; i<size; i++){
        if (arr[i] == 0) zeroCount++;
        if (arr[i] == 1) oneCount++;
    }

    cout<<"Total Number of Zero is " <<zeroCount<<endl;
    cout<<"Total Number of One is " <<oneCount<<endl;    
}

int main(){

    int myArr[10] = {0, 1, 1, 0, 1, 0, 1, 1, 0, 2};

    int size = 10;

    zeroOne(myArr, size);

    return 0;
}

File: Python/Arrays/03.ZeroesOnes.py
def zeroOne (array):
    zeroCount = 0
    oneCount = 0

    for i in range (len(array)):
        if array[i] == 0:
            zeroCount +=1

        if array[i] == 1:
            oneCount +=1

    print("Total number of Zero in array is ", zeroCount)
    print("Total number of One in array is ", oneCount)


arr = [0, 1, 1, 0, 1, 0, 1, 1, 0, 2]

zeroOne(arr)